### PR TITLE
Try to use AutomationProperties.AutomationId for VisualTreeItem name if element name is not specified.

### PR DESCRIPTION
**Pull request by paulspiteri**

### CODE REVIEW

**Comment by raybooysen** [comment]
(shipit)

File: Snoop/VisualTreeItem.cs [hunk 1]
```diff
@@ -179,9 +180,17 @@ public virtual bool HasBindingError
 		/// </summary>
 		public void Reload()
 		{
-			this.name = (this.target is FrameworkElement) ? ((FrameworkElement)this.target).Name : string.Empty;
-
-			this.nameLower = (this.name ?? "").ToLower();
+            var targetFrameworkElement = target as FrameworkElement;
```
Comment (paulspiteri) [inline Snoop/VisualTreeItem.cs]: simple comment

**Comment by cplotts** [comment]
lol. Almost closed this one because it was so old ... but I like it.

**Comment by raybooysen** [comment]
@cplotts Thanks! Patience is a virtue. :D

**Comment by cplotts** [comment]
@raybooysen I review things very very closely :joy: ... it takes a while! :100:

**Pull request merged**